internal/codegen/descriptorcodegen/testdata/1.sql.database.go

```diff
@@ -23,19 +23,19 @@ var descriptor = databaseDescriptor{
 		},
 		firstName: columnDescriptor{
 			columnID:             "FirstName",
-			columnType:           spansql.Type{Array: false, Base: 3, Len: 1024},
+			columnType:           spansql.Type{Array: false, Base: 4, Len: 1024},
 			notNull:              false,
 			allowCommitTimestamp: false,
 		},
 		lastName: columnDescriptor{
 			columnID:             "LastName",
-			columnType:           spansql.Type{Array: false, Base: 3, Len: 1024},
+			columnType:           spansql.Type{Array: false, Base: 4, Len: 1024},
```

Comment:
Did they change the numbering of types? Should this be considered a breaking change?

Reply:
It seems like it - we need to make sure to bump Spanner and this library together.

Reply:
Would we be more "safe" if we used the const values instead of the "numeric" value?
I'm actually not sure how golang handles multiple version of the same dependency

Reply:
The oldest possible dependency will be used, I think: https://about.sourcegraph.com/blog/the-pain-that-minimal-version-selection-solves

Reply:
I hoped to avoid re-implementing a String function for these enums, but I agree it's the right thing to do to ensure we keep things stable over versions.

I'll push another commit that fixes this.